linkatos/parser.py

```diff
@@ -27,4 +27,17 @@ def parse_reaction_added(event):
         'user': event['user'],
         'to_user': event['item_user']
     }
+
     return reaction
+
+
+def parse_list_request(event):
+    if not message.is_list_request(event['text']):
+        return {'type': None}
```

Comment:
I would return `None` here and then check for `if 'type' in res and res['type'] == 'foo'`.